src/cdk/menu/menu-base.ts

```diff
@@ -141,6 +141,20 @@ export abstract class CdkMenuBase
     this.keyManager.setLastItemActive();
   }
 
+  /**
+   * Sets the active item to the item at the specified index and focuses the newly active item.
+   * @param item The index of the item to be set as active, or the CdkMenuItem instance.
+   */
+  setActiveMenuItem(item: number | CdkMenuItem) {
+    if (this.keyManager) {
+      if (typeof item === 'number') {
```

Comment:
You don't need this check, the key manager does it automatically.

Reply:
Yes its not needed but typescript is not happy so I had to add.

Reply:
I see. Let's keep it as is and I'll clean it up later.